### DISCUSSION TITLE
Is it possible to have a dynamic snippet that will be a link to the parent folder?

### DISCUSSION THREAD

**Discussion by bblackmoor** [SilentVoid13/Templater#1209] [Help] [answered]
Is it possible to have a dynamic snippet that will be a link to the parent folder?

Example: I am looking at document Recipes, which is in the folder Notes. I would like to paste a snippet at the beginning of the Recipes document, so that if I click it, it will take me up to Notes (technically Notes/Notes.md, but I believe that A's Folder Notes handles that redirection... or maybe it would be more efficient to code the path directly, hmm).

Thank you.

**Comment by Zachatoo** [comment, marked as answer]
Although technically possible with [dynamic commands](https://silentvoid13.github.io/Templater/commands/dynamic-command.html), they are planned to be deprecated (#913).

I know that Dataview can definitely do this, and is a much better solution for anything that needs to be dynamic and not static.
